src/lib/Navigation.svelte

```diff
@@ -10,6 +89,29 @@
 			<div class="mzp-c-navigation-logo">
 				<a href="/">data@mozilla</a>
 			</div>
+			<nav class="mzp-c-menu mzp-is-basic">
+				<ul class="mzp-c-menu-category-list">
+					{#each navigations as navigation}
+						<li class="mzp-c-menu-category mzp-has-drop-down mzp-js-expandable">
+							<a class="mzp-c-menu-title disabled" href="/#" aria-haspopup="true"
+								>{navigation.name}</a
+							>
+							{#if navigation.items && navigation.items.length}
+								<div class="mzp-c-menu-panel mzp-c-emphasis-box" id="mzp-c-menu-panel-example">
```

Comment:
Why `-example` for the `id`?

Reply:
forgot to remove this part. thanks for catching it!